src/containers/Homepage/types.tsx

```diff
@@ -0,0 +1,18 @@
+export interface Props {
+    
+    classes: any;
+    hourlytransactions: Transactions[];
+    isError: boolean;
+    message: string;
+    loadHourlyTransactions: (date: number) => void;
+    initMessage: () => void;
+}
+
+export interface States {
```

Comment:
if stays empty i don't think is necessary

Reply:
Fixed